src/DoctrineMiddleware.php

```diff
@@ -113,6 +113,7 @@ public function setup()
         if (!$this->setupMetadataDriver($config)) {
             throw new \RuntimeException('No Metadata Driver defined');
         }
+        $config->setAutoGenerateProxyClasses($this->getOption('auto_generate_proxies', true) ? true : false);
```

Comment:
Can you tell me what benefit we're gaining by doing it that way?

My thought process for `$this->getOption('auto_generate_proxies', true) ? true : false` was margin of error; if somebody were to put 1 or 0 as their `auto_generate_proxies` config value, the ternary would resolve to the correct boolean value where the triple-equals would not.